frontend/src/scenes/sceneLogic.ts

```diff
@@ -236,20 +241,37 @@ export const sceneLogic = kea<sceneLogicType>({
         ],
     },
     urlToAction: ({ actions }) => {
+        const unmountFeatFlagLogic = featureFlagLogic.mount()
+        const featFlags = featureFlagLogic.values.featureFlags
+
         const mapping: Record<string, (params: Params) => any> = {}
 
+        let experimentRedirect: string | undefined = undefined
```

Comment:
With this new setup, I will still be unable to reach the new home page until I complete the onboarding process (whether or not I ingest events), is this expected?

Reply:
let's chat about this during our sync today i have some questions around the current onboarding